<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminmenu</class>
 <widget class="QDialog" name="adminmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lib300::SuperUser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
backgroundcolor:#85, 87, 83
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>1251</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="book">
    <widget class="QTableWidget" name="book_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>701</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 italic 9pt &quot;Noto Sans&quot;;
</string>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Author    </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Genre     </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Rating        </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Publisher    </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Year    </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Pic</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Status</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>10</y>
       <width>447</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="soerYear">
        <property name="text">
         <string>Sort by year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortGenre">
        <property name="text">
         <string>Sort by genre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortName">
        <property name="text">
         <string>Sort by name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortStatus">
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>Sort by status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortAuth">
        <property name="text">
         <string>Sort by author</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="picBook">
     <property name="geometry">
      <rect>
       <x>753</x>
       <y>80</y>
       <width>181</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="read">
    <widget class="QPushButton" name="statusbtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="readers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>401</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Noto Sans&quot;;
</string>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Surname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Card Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>      Status</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>11</y>
       <width>711</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="snam">
        <property name="text">
         <string>Sort by name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scard">
        <property name="text">
         <string>sort by cardnumber</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sstats">
        <property name="text">
         <string>sort by status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="ord">
    <widget class="QTableWidget" name="orderss">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>331</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Noto Sans&quot;;
</string>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>bookId</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>userId</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>721</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Show active orders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="accept">
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decline">
        <property name="text">
         <string>Decline</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="NameL">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>77</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pic">
     <property name="geometry">
      <rect>
       <x>523</x>
       <y>120</y>
       <width>191</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="orby">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>120</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordered by:</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameo">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>120</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="surneo">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>120</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="carby">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>150</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Card id:</string>
     </property>
    </widget>
    <widget class="QLabel" name="curdeo">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>150</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="searchbookLE">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>821</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readersBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Readers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bookBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Books</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Orders</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 154, 6);</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(164, 0, 0);</string>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 101, 164);</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 160, 0);</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login:</string>
   </property>
  </widget>
  <widget class="QLabel" name="login">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="role">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
